platform-tests/src/test/java/org/junit/platform/launcher/core/LauncherFactoryTests.java

```diff
@@ -333,6 +338,29 @@ public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult
 		}));
 	}
 
+	@Test
+	void extensionCanReadValueFromSessionStore() {
```

Comment:
I think we should also test that an extension can store a value in the session-level store that can be read by a `LauncherSessionListener` in `launcherSessionClosed()`.